sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -189,7 +189,10 @@ export class ApplicationStudentsController extends BaseController {
       );
       if (notValidDates) {
         throw new UnprocessableEntityException(
-          `${INVALID_STUDY_DATES} ${notValidDates}`,
+          new ApiProcessError(
+            `${INVALID_STUDY_DATES} ${notValidDates}`,
```

Comment:
`INVALID_STUDY_DATES` and  `APPLICATION_DATE_OVERLAP_ERROR` are different right? I believe `INVALID_STUDY_DATES` is when a student request a PIR and the selected study period difference is not in between 42 days and 365 days. the controller will throw this exception